.typescript/tsbuild-cjs.json

```diff
@@ -2,7 +2,7 @@
     "extends": "./../tsconfig.json",
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es2015",
+        "target": "es5",
```

Comment:
I thought you said the target change is not necessary after re-organizing the dependencies of required libraries?